src/main/java/dk/kb/present/View.java

```diff
@@ -63,6 +65,7 @@ public View(YAML conf, String originOfCollection) {
         conf = conf.getSubMap(id);
         String[] mimeTokens = conf.getString(MIME_KEY).split("/", 2);
         mime = new MediaType(mimeTokens[0], mimeTokens[1]);
+        strategy = conf.getString(STRATEGY_KEY, "none");
```

Comment:
There are a few and clearly defines strategies, so defining them as an `enum` is the best ... strategy. This is also a clean way to describe what the different strategies are about, via Javadoc.

Reply:
There's quite a lot of _silent knowledge_ in the Javadoc for `MANIFESTATION`. Without turning it into Moby Dick, could you mention that it is oriented towards Preservica and what the DeliverableUnit/Manifestation relationship is?